polkadot/runtime/westend/src/xcm_config.rs

```diff
@@ -274,7 +274,7 @@ impl pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	// Note that this configuration of `SendXcmOrigin` is different from the one present in
 	// production.
-	type SendXcmOrigin = xcm_builder::EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
+	type SendXcmOrigin = xcm_builder::EnsureXcmOrigin<RuntimeOrigin, LocalPalletOriginToLocation>;
```

Comment:
same new conversion is different from old one, and it might break some current testing.

Reply:
updated

Reply:
it is only updated for rococo, no?

not necessary for westend?